Experimental/DistributedAuthoritySample/Assets/Scripts/Player/CameraControl.cs

```diff
@@ -0,0 +1,127 @@
+using System.Collections;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using Unity.Cinemachine;
+using Unity.Netcode;
+
+namespace Unity.Multiplayer.Samples.SocialHub.Player
+{
+    public class CameraControl : MonoBehaviour
```

Comment:
Can remove the public here.

Most of these fields need to be renamed to follow code conventions, and usages of private need to be removed.